pages/cyberia.md

```diff
@@ -1,36 +1,44 @@
 ---
 layout: simple-page
-title: Cyberia Camp 2019
-header: Cyberia Camp 2019
+title: Cyberia Camp 2020
+header: Cyberia Camp 2020
 permalink: /events/cyberia/
 breadcrumb: Cyberia Camp
 redirect_from:
- - /cyberia/
+  - /cyberia/
 ---
 
-![cyberia2019Logo]({{site.baseurl}}/images/cyberia.jpg)
+![cyberia2020Poster]({{site.baseurl}}/images/cyberia.png)
 
-Cyberia 2019 is an annual Adobe Photoshop boot camp organized by NUSSU commIT, with the aim of imparting graphic design knowledge and skills to NUS undergraduates to strengthen their skill set. You will learn basic to intermediate **Adobe Photoshop** and their practical applications through trainings by **certified** professional Adobe Photoshop trainer and a series of fun and exciting **games**. At the end of the camp, you will have the opportunity to put all the skills you learn to test by coming out an artwork in a **competition** and win attractive prizes!
+Cyberia 2020 is an annual Adobe Photoshop boot camp organized by NUSSU commIT, which aims to impart graphic design knowledge and skills to the participants to strengthen their skill set.
+You will learn Adobe Photoshop with **training conducted by a certified professional Adobe Photoshop trainer** and **a series of challenging and exciting games**.
+At the end of the camp, you will have the opportunity to put all the skills you learn to test by producing an artwork in our **competition** and stand to **win attractive prizes**!
 
 **As a participant, you will receive:**
-* Comprehensive Adobe Photoshop training + Certificate **(worth $450)**
-* Camp T-shirt
-* Accommodation & meal
 
-**Details of Cyberia 2019:**
-* **Date:** 11th - 13th December 2019
-* **Camp fee:** **$60** (worth $450)
+- Comprehensive Adobe Photoshop training + Certificate **(worth \$450)**
+- Camp T-shirt
+- Goodie bag
 
-Note: The allocation of spaces are based on first-come-first-served basis based on the payment of camp fee. Your RSVP will only be confirmed upon completion of payment of camp fee. **No refund can be made upon payment.**
+**Details of Cyberia 2020:**
+
+- **Date:** 7th - 9th December 2020
+- **Camp fee:** $30 **(worth $450)**
+- **Venue:** Fully online
+
+**Note**: The allocation of spaces are based on first-come-first-served basis based on the payment of camp fee. Your RSVP will only be confirmed upon completion of payment of camp fee. **No refund can be made upon payment.**
 
 Do look up for emails once you completed the RSVP and the payment!
 
-Sign up now at http://bit.ly/cyberia19 to upskill yourself! See you in the camp!
+Sign up now at [http://bit.ly/Cyberia_2020](http://bit.ly/Cyberia_2020) to upskill yourself! See you in the camp!
+
+<!-- Additional Information (?) -->
+<!-- # Section 2.1 - Additional Information & Sneak Peak & Main Event Page Redirect Link -->
+<!-- May include redirect link to event website and sneakpeak, or past photo -->
```

Comment:
You will add this in a separate PR?

Reply:
This will not be included, so ill be removing it.